ivy/func_wrapper.py

```diff
@@ -302,6 +302,46 @@ def _check_in_nested_sequence(sequence, value=None, _type=None):
 # ---------------#
 
 
+def get_decorators(fn_name, decorator_lookup):
+    """Gets the decorators from the lookup table
+    
+    Parameters
+    ----------
+    fn_name : str
+        The name of the function to get the decorators for
+    
+    decorator_lookup : dict
+        The lookup table of frozen sets of function names to decorators list
+    """
+    for funcs, decorators in decorator_lookup.items():
+        if fn_name in funcs:
+            return decorators
+    return []
```

Comment:
```suggestion
    raise KeyError(f"Function `{fn_name}` not found in lookup table!")
```

Should we raise an error when the function name is not found in the lookup table?

Reply:
yep, along with a message asking them to add the function to that list, thanks!